keyboards/ungodly/launch_pad/rules.mk

```diff
@@ -0,0 +1,33 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+MIDI_ENABLE = yes
+OLED_DRIVER_ENABLE = yes
+ENCODER_ENABLE = yes
+RGB_MATRIX_ENABLE = WS2812
```

Comment:
```suggestion
RGB_MATRIX_ENABLE = yes
RGB_MATRIX_DRIVER = WS2812
```

Reply:
This causes compiling issues.

```
quantum/rgb_matrix_drivers.c:33:21: error: 'DRIVER_ADDR_1' undeclared (first use in this function); did you mean 'TIMER_AVR_H'?
     IS31FL3731_init(DRIVER_ADDR_1);
                     ^~~~~~~~~~~~~
                     TIMER_AVR_H
quantum/rgb_matrix_drivers.c:33:21: note: each undeclared identifier is reported only once for each function it appears in
quantum/rgb_matrix_drivers.c: In function 'flush':
quantum/rgb_matrix_drivers.c:87:35: error: 'DRIVER_ADDR_1' undeclared (first use in this function); did you mean 'TIMER_AVR_H'?
     IS31FL3731_update_pwm_buffers(DRIVER_ADDR_1, 0);
                                   ^~~~~~~~~~~~~
                                   TIMER_AVR_H
 [ERRORS]
 |
 |
 |
make[1]: *** [tmk_core/rules.mk:386: .build/obj_ungodly_launch_pad_default/quantum/rgb_matrix_drivers.o] Error 1
Make finished with errors
make: *** [Makefile:584: ungodly/launch_pad:default] Error 1
```

Reply:
Update your repo.
https://github.com/qmk/qmk_firmware/pull/10231